config/locales/hyrax.en.yml

```diff
@@ -1,4 +1,5 @@
 en:
+  parents: "Parents"
```

Comment:
lets try to keep with the hyrax way.. this belongs somewhere under
```
hyrax:
  base:
    relationships_parent_rows_label: "Parents:"
```

Parents should have the trailing semi-colon so it looks like the admin set labe.

Reply:
also.. add a corresponding `.........rows_empty_text: "This work has no parents."`